web/src/components/MomentaryChat.tsx

```diff
@@ -20,47 +20,58 @@ const ReactionButton = React.memo<{
 const MomentaryChatContent = React.memo<{
   chatList: ChatList;
   replyChat: ReplyChat;
-}>(({ chatList, replyChat }) => (
-  <ul className="MomentaryChat-list">
-    {chatList.map((item) => {
-      const reply = (text: string) => replyChat(text, item.replyTo);
-      return (
-        <li key={item.key} className="MomentaryChat-listPart">
-          <div className="MomentaryChat-listPart-header">
-            <div className="MomentaryChat-iconButton-container">
-              <div className="MomentaryChat-iconButton">
-                {reactions.map((text) => (
-                  <ReactionButton key={text} text={text} onClick={reply} />
-                ))}
+  onUpdateLayout: (height: number) => void;
+}>(({ chatList, replyChat, onUpdateLayout }) => {
+  const chatListRef = useRef<HTMLUListElement | null>(null);
```

Comment:
```suggestion
  const chatListRef = useRef<HTMLUListElement>();
```

We prefer this pattern.